packages/bootleg-common/bin/bootleg-truffle-test

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Exit script as soon as a command fails.
+set -o errexit
```

Comment:
I've never seen this before. Is it similar to `set -e`?

Reply:
Honestly, I stole the code from https://github.com/0xProject/dev-tools-truffle-example/issues/5#issuecomment-457290796 after seeing that it was clearly better than the crappy bash I'd written originally. THEN, I didn't seak to understand the bits as well 😫 Shame on me. 

https://www.gnu.org/software/bash/manual/bash.html#The-Set-Builtin

Turns out this is equivalent to `set -e` 🤷‍♂️

Reply:
Ah okay, it seems more specific of an exit condition than ‘-e’ but I was not sure. Thanks for investing